packages/xforms-engine/src/lib/reactivity/createItemCollection.ts

```diff
@@ -25,8 +25,8 @@ type DerivedItemLabel = ClientTextRange<'item-label', 'form-derived'>;
 
 const derivedItemLabel = (context: TranslationContext, value: string): DerivedItemLabel => {
 	const chunk = new TextChunk(context, 'literal', value);
-
-	return new TextRange('form-derived', 'item-label', [chunk]);
+	//TODO: work this scenario
+	return new TextRange(null, 'form-derived', 'item-label', [chunk], null);
```

Comment:
@lognaturel Does `derived label` ring a bell? A label can be missing (L38), and then this derived label is created. I wonder if it can contain media.